docs/source/examples/example_dates.rst

```diff
@@ -0,0 +1,158 @@
+Example: Dates
+==============
+
+This example concerns itself with expressing ``Constraint`` s against data revolving
```

Comment:
Why do you tend to have a space character when making plurals of "formatted" words? Like here in `` Constraint`` s?

I saw the rendered version, and I think I can see a space between the word and the last letter.

Reply:
> I saw the rendered version, and I think I can see a space between the word and the last letter.

Same here - I don't like the look of it.

Afaict it is not valid rst to add a character right after a closing double-tick. 

If I do, the build gets very messy:

![Screenshot 2022-09-19 at 10 44 07 AM](https://user-images.githubusercontent.com/7267523/190980858-50b78ffe-5795-4c33-84f1-f112087b0d8e.png)